docs/_documentations/metrics-dashboard.md

```diff
@@ -43,39 +43,32 @@ View these metrics to learn if your application is performing as intended. An ap
 * **Loop Times:** This graph indicates how fast a Node.js project is processing through an event loop.
 * **Other Requests:** This graph indicates the Socket.IO and other requests that the application processes.
 
-## Clicking the **Run load** button
-
-Click the **Run load** button to run load your microservice. This button pings endpoints by using the Codewind load runner service.
-
 ## Viewing metrics information in the **Summary** tab
 
 View a summary of the information from the performance metrics graphs that you saw in the **Dashboard** tab.
 * **HTTP Requests:** This table displays how many times an endpoint is hit. If you repeatedly run a test while making code changes, you can see how the code changes are impacting application performance.
 * **Environment:** This table shows information about the environment you're running, such as the OS architecture and the number of processors you have. Having this information can make troubleshooting easier if you need to contact support and provide it.
-* **Resource usage:** This table shows the **Average Process CPU**, which shows what Codewind is doing, and the **Average System CPU**, which shows what the environment as a whole is doing. This information can help determine whether application issues are caused by something else in the environment that is external to the application.
+* **Resource Usage:** This table shows the **Average Process CPU**, which shows what Codewind is doing, and the **Average System CPU**, which shows what the environment as a whole is doing. This information can help determine whether application issues are caused by something else in the environment that is external to the application.
```

Comment:
`**Average Process CPU**, which shows what Codewind is doing` is wrong, we aren't monitoring codewind's CPU usage, it's the applications CPU usage.

Reply:
Corrected